internal/pkg/utils/testproject/snapshot.go

```diff
@@ -201,6 +216,23 @@ func (p *Project) NewSnapshot() (*fixtures.ProjectSnapshot, error) {
 		}
 	}
 
+	// Join sandbox instances with config name
+	for _, config := range configsMap {
+		if config.ComponentID == sandboxesapi.Component {
+			sandboxId, err := sandboxesapi.GetSandboxID(config.ToApi().Config)
```

Comment:
Toto by sa malo diat paralelne, resp. nacitat vsetko naraz jednym callom.
Alebo je tam nejaky problem?
(ale to nebude asi ten problem s testami)

Reply:
aha ok, dik za vysvetlenie